From 6a9f85f9050e2c0f38148e295d7e25f56d05c6de Mon Sep 17 00:00:00 2001
From: AppaRao Puli <apparao.puli@linux.intel.com>
Date: Mon, 1 Feb 2021 23:45:53 +0000
Subject: [PATCH] Fix wrong check in EventService events

Sending async event logs to event listener is
broken due to commit 23a21a1cbed23ace4174664950e595df961e9e69.
Correct the check to make EventService back to functional
state.

Tested:
 - Redfish event logs are properly sent to subscribers.

Signed-off-by: AppaRao Puli <apparao.puli@linux.intel.com>
Change-Id: If232846a2b0ac694205731a801e55dc4bd5e928a
---
 redfish-core/include/event_service_manager.hpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/redfish-core/include/event_service_manager.hpp b/redfish-core/include/event_service_manager.hpp
index 54dafb4..3db9f0c 100644
--- a/redfish-core/include/event_service_manager.hpp
+++ b/redfish-core/include/event_service_manager.hpp
@@ -1208,7 +1208,7 @@ class EventServiceManager
 
     static void watchRedfishEventLogFile()
     {
-        if (inotifyConn)
+        if (!inotifyConn)
         {
             return;
         }
-- 
2.17.1

